### PR TITLE
JEN-1066 add ubuntu 'disco' to PS jenkinses

### PR DESCRIPTION
**Pull request by hors**

### CODE REVIEW

File: jenkins/pipeline.groovy [hunk 1]
```diff
@@ -25,7 +25,7 @@ pipeline {
             description: 'Tag/Branch for Percona-TokuBackup repository',
             name: 'TOKUBACKUP_BRANCH')
         choice(
-            choices: 'ubuntu:artful\ncentos:6\ncentos:7\ni386/centos:6\nubuntu:trusty\nubuntu:xenial\nubuntu:bionic\nubuntu:cosmic\ndebian:jessie\ndebian:stretch',
+            choices: 'ubuntu:artful\ncentos:6\ncentos:7\ni386/centos:6\nubuntu:trusty\nubuntu:xenial\nubuntu:bionic\nubuntu:cosmic\nubuntu:disco\ndebian:jessie\ndebian:stretch',
```
Comment (EvgeniyPatlan) [inline jenkins/pipeline.groovy]: We should remove artuful as it is unsupported OS for now so no need to run tests on it

File: jenkins/trunk.yml [hunk 2]
```diff
@@ -145,6 +145,7 @@
           - ubuntu:artful
```
Comment (EvgeniyPatlan) [inline jenkins/trunk.yml]: the same

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: jenkins/pipeline.groovy (hunk 1 above)
Comment (hors) [reply jenkins/pipeline.groovy]: done

File: jenkins/trunk.yml (hunk 2 above)
Comment (hors) [reply jenkins/trunk.yml]: done

**Review by EvgeniyPatlan — APPROVED** [review]

**Pull request merged**